FCCee/IDEA/compact/IDEA_o1_v02/IDEA_o1_v02.xml

```diff
@@ -52,6 +52,9 @@
 
   <!-- Import Endcap plate absorber -->
   <include ref="EndPlateAbsorber_o1_v01.xml"/>
+  
+  <!-- Import fiber-based dual-readout calorimeter -->
+  <include ref="FiberDualReadoutCalo_o1_v01.xml"/>
```

Comment:
I think we only need to push to the v03, not v02 (as we discussed in the working meeting)